services/ui-auth/serverless.yml

```diff
@@ -178,12 +175,6 @@ resources:
     CognitoAuthRole:
       Type: AWS::IAM::Role
       Properties:
-        Path: ${self:custom.iamPath}
-        PermissionsBoundary:
-          Fn::If:
-            - CreatePermissionsBoundary
-            - !Sub arn:aws:iam::${AWS::AccountId}:policy${self:custom.iamPermissionsBoundaryPolicy}
-            - !Ref AWS::NoValue
```

Comment:
We can now rely on the serverless-iam-helper plugin to handle our IAM Path/Perm requirements.